File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_bankwdrwrn.java
// EXPECTED: 155.00
// Reference emit for BANKWDRWRN — 200 - 45 remaining.
public final class BankWdrwrnRef {
  public static void main(String[] args) {
    double bal = 200.00;
    double wdrw = 45.00;
    double remain = wdrw <= bal ? Math.round((bal - wdrw) * 100.0) / 100.0 : bal;
    System.out.printf("%.2f%n", remain);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_bankwdrwrn.py
"""Reference emit for BANKWDRWRN — rocket-bank withdrawal remaining."""
# EXPECTED: 155.00
BAL = 200.00
WDRW = 45.00
REMAIN = round(BAL - WDRW + 1e-12, 2) if WDRW <= BAL else BAL
print(f"{REMAIN:.2f}")
